test/cloudtest/setup

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+set -euo pipefail
+
+cd "$(dirname "$0")/../.."
+
+. misc/shlib/shlib.bash
+
+run kind create cluster --name=cloudtest --config=misc/kind/cluster.yaml --wait=60s || true
```

Comment:
@benesch I had to re-introduce the `|| true` on this line, as in CI `kind create cluster` will fail if the cluster already exists. So the non-zero exit code needs to be eaten, as this entire script runs under `set -e`

Reply:
Ahh, that makes sense. Too bad there is no create-if-not-exists option. This has the potential to hide real errors during startup.